src/app/services/auth.service.ts

```diff
@@ -57,7 +57,7 @@ export class AuthService {
       );
   }
 
-  private hashPassword(form: LoginForm | RegisterForm) {
+  private static hashPassword(form: LoginForm | RegisterForm) {
```

Comment:
I can't say that it is wrong. But I don't understand the reason. When we make object static this means that this object will placed in memory all time. I think we can left this method as is.

Reply:
Agree. No need